packages/common/src/types/sqlRunner.ts

```diff
@@ -14,3 +20,71 @@ export type SqlRunnerBody = {
 export type SqlRunnerResults = ResultRow[];
 
 export const sqlRunnerJob = 'sqlRunner';
+
+export type SqlChart = {
+    savedSqlUuid: string;
+    name: string;
+    description: string | null;
+    slug: string;
+    sql: string;
+    config: object;
+    chartKind: ChartKind;
+    createdAt: Date;
+    createdBy: Pick<
+        LightdashUser,
+        'userUuid' | 'firstName' | 'lastName'
+    > | null;
+    lastUpdatedAt: Date;
+    lastUpdatedBy: Pick<
+        LightdashUser,
+        'userUuid' | 'firstName' | 'lastName'
+    > | null;
+    space: Pick<Space, 'uuid' | 'name'>;
+    dashboard: Pick<Dashboard, 'uuid' | 'name'> | null;
+    project: Pick<Project, 'projectUuid'>;
+    organization: Pick<Organization, 'organizationUuid'>;
+};
+
+export type CreateSqlChart = {
+    name: string;
+    description: string | null;
+    sql: string;
+    config: object;
+    spaceUuid: string;
+};
+
+export type UpdateUnversionedSqlChart = {
+    name: string;
+    description: string | null;
+    spaceUuid: string;
+};
+
+export type UpdateVersionedSqlChart = {
+    sql: string;
+    config: object;
+};
+
+export type UpdateSqlChart = {
+    unversionedData?: UpdateUnversionedSqlChart;
+    versionedData?: UpdateVersionedSqlChart;
+};
+
+export type ApiSqlChart = {
+    status: 'ok';
+    results: SqlChart;
+};
+
+export type ApiCreateSqlChart = {
+    status: 'ok';
+    results: {
+        savedSqlUuid: string;
```

Comment:
This is all we need, does it need a type? I just stubbed it out like this in the API call

Reply:
Would it return the same chart kind type as the one sent in the body? My idea was for the FE to change the URL and fetch the saved SQL with the GET endpoint.